### ISSUE TITLE
[FE] Profile: In the "nametag" element, what happens if someone is in Space Force?

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#40910]
### Background

We need to accommodate showing military info for Space Force veterans. While I'm pretty sure that would work just fine in the actual "military information" section of the profile if someone was in Space Force, I'm not 100% sure what happens in the nametag. I know we do not currently have an emblem for Space Force, but can we at least show the branch under the veteran's name?

![image](https://user-images.githubusercontent.com/34068740/166801260-68854c59-f8ee-4420-bb0d-d02fa3dfa6b0.png)

### Task

- [ ] Determine what someone sees if they are looking at their nametag and their branch of military service is the Space Force.

### Acceptance criteria

- [ ] We understand how the nametag behaves for Space Force veterans
- [ ] If there are updates that need to be made to the nametag, note them in this ticket

**Comment by mdewey** [comment]
## TL;DR;

 If the branch is unknown, in this case, Space Force, the name card defaults to not showing a badge but still displaying the name (See screenshot). So it at least displays the branch name

![Screen Shot 2022-05-24 at 9 46 24 AM](https://user-images.githubusercontent.com/1793923/170050923-945b2e01-d7c0-42ea-bf08-042f3d53281a.png)

## Next steps

To get the full experience for a Space Force Veteran: 

- Determine what the constant for `Space Force` is that is being returned from the profile API
- Obtain an emblem for Space Force. 

## More Details

- We have a set of pre-determined [constants](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/personalization/profile/constants.js#L3). If the veteran's branch name is found, we display `United States XXXXX`, else we display only the branch name
- To add a new branch, we need to add a new constant and a new picture. We can add a new constant without adding a new picture. This is set up to fail gracefully.

**Comment by Samara-Strauss** [comment]
Thank you so much! This is super helpful. No further steps needed. I just wanted to make sure that this element wouldn't break for Space Force folks. Since it doesn't, we're good.
